pkg/csi/hostpathimpl.go

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright (c) 2023 ApeCloud, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	log "k8s.io/klog/v2"
+
+	"github.com/alibaba/open-local/pkg"
+)
+
+const (
+	hostPathBasePathEnv = "HOSTPATH_BASE_PATH"
+
+	// tags from StorageClass
+	hostPathScTag             = "hostPath"
```

Comment:
what about add a prefix like "csi.openlocal.kubeblocks.io/"

Reply:
These fields are placed in `storageClass.parameters`, indicating that they are specific to this driver, so there is no need to add a prefix. If they are in annotations, it is necessary to add one.